.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - "++2.12.10 testsJVM/test"
           - "++2.12.11 testsJVM/test"
           - "++2.12.11 testsJS/test"
+          - "++2.12.12 testsJS/test"
```

Comment:
looks like we run JVM tests for all versions and JS only for latest.
Suggestion if that is the case:
remove `"++2.12.11 testsJS/test"`
add `"++2.12.12 testsJVM/test"`